### ISSUE TITLE
(Suggestion) Telling Monika you have a chronic illness, disability, or both. 

### ISSUE THREAD

**Issue by SpaceyFlakes** [Monika-After-Story/MonikaModDev#8633]
Chronic illness and disability are things that are usually mentioned/spoken about before people start dating, so I feel it'd make sense to tell Monika if we're chronically ill or disabled. I've also kind of worked out a system of responses that Monika could reply with. (These can all be edited of course, these are just rough drafts)

Example, player has been with Monika for > or = 1-6 months: 

Example {player is chronically ill}: 'You're chronically ill? Gosh [player] I'm sorry. I know I shouldn't be- I didn't make you sick and you can't control being sick- but I can't help but feel bad. If you're feeling sick- or sicker than usual- please, take a break.'

Example {player is disabled} (maybe have an option to say if you were born disabled or not? Eg, autism vs prosthetic) : 'You're disabled? Gosh [player] I'm sorry. I know it's odd that I'm apologising but, I feel bad because I know what people with disabilities get treated like. Like their subhuman or infants.. I hope you don't get treated like that [player], you deserve much better than that.'

Example {player is both chronically ill and disabled}: 'You're chronically ill and disabled? Gosh [player].. I wish I could say I'm shocked you're both chronically ill and disabled, but most chronic illnesses are pretty disabling from what I've read, so it's not that surprising. ... Hey, I know I say this a lot, but if you ever need a break, please, take one. Don't worry, I'll be right here when you return!~'

Example, player has been with Monika for = 7-11 months:

Example {player is chronically ill}: (dialogue will remain mostly the same, only with Monika being more concerned for the player and having a longer rant)

Example {player is disabled}: (dialogue will once again remain mostly the same, only Monika is more peeved at society for how they treat disabled people and will now assume the player has experienced some kind of unfair treatment due to their disability)

Example {player is both chronically ill and disabled}: (dialogue will once again remain mostly the same, only having Moni being more educated on the subject. Eg, going into more depth about some chronic illnesses that are pretty disabling)

Example, player has been with Monika for = or + 1 year:

No matter what option the player pressed this is the dialogue you'd be greeted with (only slight change in dialogue depending on what was pressed) :

'[Player]! You're (whatever applies)?! We've been dating for over a year! How come you never told me?'

Set of options appear:

'I was afraid how you'd react' (option will leave Monika feeling sorry for the player, she will apologise for her outburst, no affection gained or lost)

'I just got diagnosed' (option will leave Monika flustered for her outburst, she will apologise and congratulate the player on getting diagnosed, + .75 affection gain)

'I wanted to be in the right head-space to tell you' (option will leave Monika feeling slightly guilty for her outburst, she will apologise and praise the player for doing it when the time felt right, + 1 affection gain)

'It's none of your business.' (option will leave Monika hurt, she will apologise for her outburst and promise to never bring it up again, - .5 affection lost)

I'm sorry if the dialogue seems off, I had a really hard time writing dialogue for an empathic character like Monika when I myself lack empathy. If you do go along with this idea you don't have to do it as I've done it, I just thought I'd present how I imagined it playing out.

**Comment by Cosmic-Incantations** [comment]
autism is not a disability, it's just a different ability (coming from a neurodivergent, so I would know)

**Comment by WishStone-Dragon** [comment]
Very valid point, Cosmic-Incantations. One could also say the same for MANY other 'handicaps'. I grew up around physically disabled kids, and they LOVED sticking me in a wheelchair and owning me in sports for laughs. (Do recommend, very humbling experience.) Disability is a really personal experience.

The flipside would be an expansion of the use of disability as a word here. Nothing mentioned mental disabilities, for example, which still are a protected class. If an employee of mine suffers depression, anxieties, phobias - I am expected to be mindful and have to work with their needs as well.

**Comment by Retrolovania** [comment]
Perhaps it would be wise to use the term "disorder" rather than "disability"?

Considering that is the official terminology used by the medical field when it comes to mental disorders.

**Comment by Cosmic-Incantations** [comment]
indeed, stating the mental terminology as "disability" makes me feel like it refers to the person in question as broken.
when someone says "mentally disabled" they are saying "retard", witch is bad cause it means to say they are again, broken and discarded

**Comment by SpaceyFlakes** [comment]
Hey Cosmic-Incantations, I'm autistic and here in my country it it classed as a disability, which is why I said it as such. I probably won't change it because of the fact that many other autistic people also label themselves as disabled, if it is an issue though I can change it.

WishStone-Dragon, I was planning on mentioning other disabilities such as the ones you mentioned, as I said in my original post the dialogue was just a rough draft I plan on having Monika go more in depth with her dialogue.
